packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -183,34 +183,40 @@ const generateLiveMessage = (props: ChatMessageContentProps): string => {
 };
 
 const messageContentAriaText = (props: ChatMessageContentProps): string | undefined => {
-  // Strip all html tags from the content for aria.
+  if (props.message.content) {
+    // Replace all <img> tags with 'image' for aria.
+    const parsedContent = DOMPurify.sanitize(props.message.content, {
+      ALLOWED_TAGS: ['img'],
+      RETURN_DOM_FRAGMENT: true
+    });
 
-  return props.message.content
-    ? props.message.mine
+    parsedContent.childNodes.forEach((child) => {
+      if (child.nodeName !== 'IMG') {
+        return;
+      }
+      const imageTextNode = document.createElement('div');
+      imageTextNode.innerHTML = 'image';
+      parsedContent.replaceChild(imageTextNode, child);
```

Comment:
Tested anf I'm wondering if it's possible to remove `&nbsp`values in the aria label?

Reply:
> Tested anf I'm wondering if it's possible to remove `&nbsp`values in the aria label?

Good question. I'll ask the Ally team whether they prefer that and if so, I'll create a separate ally bug for it. There's some scenarios to consider such as what if user actually enters `&nbsp`

Reply:
> &nbsp

Talked to Wayne, since this &nbsp is what we receive within the html, we would not change it. We receive this  &nbsp with or without inline image if there's white space in the message.
"messageBody\": \"<p>&nbsp;&nbsp;</p>\\r\\n<p><img alt=\\\"image\\\" src=\\\"\\\" width=\\\"173\\\" height=\\\"130\\\" id=\\\"0-canaryeus-d1-d6dba0ce6878faafa787bd8eca52e17d\\\"></p>\\r\\n<p>&nbsp; rrr</p>\"